autoscheduler/frontend/src/components/LastUpdatedAt.tsx

```diff
@@ -110,17 +110,17 @@ const LastUpdatedAt: React.FC = () => {
   }, [lastUpdated]);
 
   return (
-    <div
+    <Typography
       style={{
-        display: 'flex',
-        alignItems: 'center',
         paddingLeft: 8,
+        whiteSpace: 'nowrap',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
```

Comment:
We should probably be doing CSS class for this, but since this is the only thing in the component that would need it, I think it's not worth creating a new CSS file just for this

Reply:
Yeah that was my thought as well, but I kept it as a style since that's how it was originally (which I think is fine until we need multiple of them)